psutil/_psbsd.py

```diff
@@ -432,6 +432,22 @@ def sensors_battery():
             secsleft = minsleft * 60
         return _common.sbattery(percent, secsleft, power_plugged)
 
+    def sensors_temperatures():
+        """Return systemp temperatures"""
```

Comment:
I would rephrase this as: "Return CPU cores temperatures if available, else an empty dict."